### PR TITLE
Smokeping: add pings config

### PR DESCRIPTION
**Pull request by paulgear**
This allows customising the number of pings to be graphed.

### CODE REVIEW

**Comment by paulgear** [comment]
Should fix issue #2532.

**Comment by paulgear** [comment]
This should be a pretty simple one, @librenms/reviewers :-)

**Comment by Rosiak** [comment]
:+1:

File: includes/defaults.inc.php [hunk 1]
```diff
@@ -458,6 +458,7 @@ function set_debug($debug) {
 // Ignore lines starting with #
 // $config['collectd_dir']                 = '/var/lib/collectd/rrd';
 // $config['smokeping']['dir']             = "/var/lib/smokeping/";
+$config['smokeping']['pings']             = 10;
```
Comment (f0o) [inline includes/defaults.inc.php]: Why the change in default behavior and not keeping the `old` value like people are used to?

Comment (paulgear) [inline includes/defaults.inc.php]: Good point; will update
